test/pools/disjoint_pool_ctl.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2025 Intel Corporation
+// Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exceptiongi
+
+#include <umf/memory_pool.h>
+#include <umf/pools/pool_disjoint.h>
+
+#include "base.hpp"
+#include <umf/memory_pool.h>
+#include <umf/memory_provider.h>
+#include <umf/memory_provider_ops.h>
+#include <umf/providers/provider_os_memory.h>
+
+using umf_test::test;
+using namespace umf_test;
+
+TEST_F(test, disjointCtlName) {
+    umf_memory_provider_handle_t hProvider = NULL;
+    umf_os_memory_provider_params_handle_t os_memory_provider_params = NULL;
+    umf_memory_provider_ops_t *os_provider_ops = umfOsMemoryProviderOps();
+    if (os_provider_ops == NULL) {
+        GTEST_SKIP() << "OS memory provider is not supported!";
```

Comment:
Instead of adding such skip msg, I guess we should add this test in CMake only when OS prov and disjoint pool are enabled.

Reply:
It's what for we've added a dummy function.